### DISCUSSION TITLE
Separate directory for options and savegames

### DISCUSSION THREAD

**Discussion by mase76** [Keriew/augustus#465] [Ideas]
Hi!
Wouldn't it be better to have a separate directory for savegames and options in the user's home, instead of having it in the assets dir? Maybe ~/.config/julius. On multiuser systems each user must install the same assets in his homedir. The config file could also include the level of screen scaling, instead of adding it to the commandline. When installing the game, a desktop entry is created, but without screen scaling.

**Comment by crudelios** [comment]
Yes it would. Augustus is still too reliant on Julius in that regard, but eventually I expect a new directory structure to be implemented.

**Comment by crudelios** [comment]
Afaik, some of this has now been fixed:

* The screen scaling in now an in-game option
* The new assets can be placed in multiple different places

What's missing is a separate directory for saves and configurations. I think these could go to [`SDL_GetPrefPath`](https://wiki.libsdl.org/SDL_GetPrefPath)?

**Comment by gianlore** [comment]
Come ho già scritto in altra parte, mi viene chiesto all'avvio di augustus i file/directory "assets", nessuno ne parla.
Da me ci sono molti problemi /il teatro è un rettangolo nero, i blocchi stradali non vengono creati e altre cose.
